spec/features/admin/order_cycles_spec.rb

```diff
@@ -58,6 +58,39 @@
     end
   end
 
+  describe 'listing order cycles with other locales' do
+    let!(:oc_de) { create(:simple_order_cycle, name: 'oc', orders_open_at: '2012-01-01 00:00:00') }
+
+    around(:each) do |spec|
+      I18n.locale = :de
+      spec.run
+      I18n.locale = :en
+    end
+
+    context 'using datepickers' do
+      it "correctly open the datepicker and changes the date field" do
+        login_to_admin_section
+        visit admin_order_cycles_path
+
+        within("tr.order-cycle-#{oc_de.id}") do
+          expect(all('input').first.value).to start_with '2012-01-01 00:00:00'
```

Comment:
please, use `find(".my-selector", match: :first)` instead of `all('input').first`. The former will wait for at least one element matching selector to appear on the page, whereas the latter won't. This is a source of brittle tests.